ideascube/mixins.py

```diff
@@ -9,19 +9,43 @@
 from taggit.models import Tag
 
 
-class ByTagListView(ListView):
+class FilterableViewMixin:
+    def _search_for_attr_from_context(self, attr, context):
+        search_context = {k:context[k] for k in ('q', 'kind', 'lang', 'tags')
+                            if k != attr}
```

Comment:
Shouldn't we filter out None values, something like `if k!= attr and context[k]`?